packages/editor/src/components/document-outline/remove-url-hash.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { defer } from 'lodash';
+
+// Remove the URL hash.
```

Comment:
This could (ought) be defined as proper JSDoc of the function.

Reply:
improved in https://github.com/WordPress/gutenberg/pull/10815/commits/99380f2250a4fa2a5ba3e5b6ef5afb0f68f2de9e - no paramaters or return, do I need to add anything else?